MapboxNavigation/MBConstants.h

```diff
@@ -0,0 +1,30 @@
+#import <Foundation/Foundation.h>
+
+/**
+ Key used for constructing errors when spoken instructions fail.
+ */
+extern const _Nonnull NSErrorUserInfoKey MBSpokenInstructionErrorCodeKey;
+
+/**
+ Posted when `MBStyleManager` applies a new style triggered by change of day of time, or when entering or exiting a tunnel.
+ 
+ :nodoc:
```

Comment:
Do these types need to be excluded from the docset?

Reply:
I assumed it had to do with the linter, but after running it, I don't see any reason to exclude these types from the docset.